server/cmwell-sparql-agent/src/main/scala/cmwell/tools/data/sparql/SparqlProcessorManager.scala

```diff
@@ -476,14 +476,21 @@ class SparqlProcessorManager(settings: SparqlProcessorManagerSettings) extends A
     }
   }
 
+
   def getJobConfigsFromTheUser: Future[Set[JobRead]] = {
     logger.info("Checking the current status of the Sparql Triggered Processor manager config infotons")
     //val initialRetryState = RetryParams(2, 2.seconds, 1)
     //retryUntil(initialRetryState)(shouldRetry("Getting config information from local cm-well")) {
     safeFuture(
       client.get(s"http://${settings.hostConfigFile}${settings.pathAgentConfigs}",
-                 queryParams = List("op" -> "search", "with-data" -> "", "format" -> "json", "length" -> "100"))
-    ).map(response => parseJobsJson(response.payload)).andThen {
+                 queryParams = List("op" -> "search", "with-data" -> "", "format" -> "jsonh", "length" -> "100"))
+    ).map{
+      case response@SimpleResponse(respCode,_,_) if respCode < 300 =>
+        parseJobsJson(response.payload)
+      case failedResponse@SimpleResponse(respCode,_,_) =>
+        logger.warn(s"Failed to read config infoton from cm-well. Error code: $respCode payload: ${failedResponse.payload}")
```

Comment:
please add the header to the log output. Sometimes it's very important during debugging (e.g. to see in the server logs what happened). One of the headers is the actual server that responded.